rook/initialise.go

```diff
@@ -218,53 +217,48 @@ func Init(
 
 	if answers.Entry != "" {
 		ext := filepath.Ext(answers.Entry)
-		nameOnly := strings.TrimSuffix(answers.Entry, ext)
-		pkg.Entry = nameOnly + ".pwn"
-		pkg.Output = "gamemodes/" + nameOnly + ".amx"
+		nameWithPath := strings.TrimSuffix(answers.Entry, ext)
+		nameOnly := filepath.Base(nameWithPath)
+
+		pkg.Entry = nameWithPath + ".pwn"
+
+		if answers.PackageType == "filterscript" {
+			pkg.Output = "../" + nameOnly + ".amx"
```

Comment:
Why is it `../` in the parent directory?

Reply:
the default should be in the filterscripts folder, which should be created in this case. I also agree with Southclaws why are you going outside the current working directory?

Reply:
> Why is it `../` in the parent directory?
Is because when you creating the filterscript, the contents should be in:

```
| -- server_root
     | -- Filterscript
          | -- my-system
```
So, doing `../` would place the `.amx` outside the folder name.

Reply:
But that also can be done using just `filterscript/` instead of that.